clients/src/main/java/org/apache/kafka/clients/consumer/ConsumerConfig.java

```diff
@@ -610,6 +628,18 @@ public class ConsumerConfig extends AbstractConfig {
                                         DEFAULT_ALLOW_AUTO_CREATE_TOPICS,
                                         Importance.MEDIUM,
                                         ALLOW_AUTO_CREATE_TOPICS_DOC)
+                                .define(GROUP_PROTOCOL_CONFIG,
+                                    Type.STRING,
+                                    DEFAULT_GROUP_PROTOCOL,
+                                    ConfigDef.CaseInsensitiveValidString
+                                        .in(Utils.enumOptions(GroupProtocol.class)),
```

Comment:
You need a wider monitor, @philipnee 😛

```suggestion
                                    ConfigDef.CaseInsensitiveValidString.in(Utils.enumOptions(GroupProtocol.class)),
```

Reply:
108 char width!